Sources/AppcuesKit/Presentation/Traits/Appcues/AppcuesTargetElementTrait.swift

```diff
@@ -41,11 +41,19 @@ internal class AppcuesTargetElementTrait: BackdropDecoratingTrait {
     }
 
     func decorate(backdropView: UIView) throws {
-        metadataDelegate?.set([ "targetRectangle": try calculateRect(bounds: backdropView.bounds) ])
+        // Determine if the selector can be resolved to a view, failing the step if not.
+        let targetRect = try calculateRect(bounds: backdropView.bounds)
```

Comment:
subtle difference here, at least for now - since we want to fail on bad selector, we need to still check for the matching view here.  Goes back to the convo on error handling in the frame observer, which has not been resolved yet.

Reply:
Makes sense. Also, it doesn't look the `bounds: backdropView.bounds` is used anywhere.

Reply:
ah you are right. will clean up. thanks